content/en/continuous_integration/pipelines/custom.md

```diff
@@ -43,7 +44,7 @@ To send pipeline events programmatically to Datadog, ensure that your [`DD_API_K
    - `DD-API-KEY`: Your [Datadog API key][14].
    - `Content-Type`: `application/json`.
 
-1. Prepare the payload body by entering information about the [pipeline execution][2] in a cURL command:
+2. Prepare the payload body by entering information about the [pipeline execution][2] in a cURL command:
```

Comment:
no related but we had `1.` on every item on the list

Reply:
If you use only `1.` for a list, Markdown will dynamically update each item to its correct number. With that said, I prefer this way, as it makes maintenance easier and avoids this kind of confusion.